packages/gatsby-source-vtex/src/gatsby-node.ts

```diff
@@ -213,16 +215,35 @@ export const sourceNodes: GatsbyNode['sourceNodes'] = async (
   activity.end()
 }
 
-export const createPages = (
-  { actions: { createRedirect } }: CreatePageArgs,
-  { tenant, workspace, environment }: Options
+export const createPages = async (
+  { actions: { createRedirect }, reporter }: CreatePageArgs,
+  {
+    tenant,
+    workspace,
+    environment,
+    getRedirects = defaultGetRedirects,
+  }: Options
 ) => {
   /**
    * Create all proxy rules for VTEX Store
    * If adding a new rule, don't forget to modify ./gatsby-config.ts dev proxy
    * so it also works in dev mode
    */
 
+  const activity = reporter.activityTimer(
+    '[gatsby-source-vtex]: soucing Redirects'
```

Comment:
Did you mean source?

```suggestion
    '[gatsby-source-vtex]: sourcing Redirects'
```